Content.Server/_DV/CosmicCult/CosmicCultSystem.Finale.cs

```diff
@@ -1,6 +1,7 @@
 using Content.Server._DV.CosmicCult.Components;
 using Content.Shared._DV.CosmicCult;
 using Content.Shared._DV.CosmicCult.Components;
+using Content.Shared._DV.CustomObjectiveSummary;
 using Content.Shared.Audio;
```

Comment:
Using directive is unnecessary.
```suggestion
using Content.Shared._DV.CosmicCult.Components;
using Content.Shared.Audio;
```